tests/testdata/codegen/simple-enum-schema/schema.json

```diff
@@ -237,6 +237,54 @@
           "value": 12
         }
       ]
+    },
+    "plant::ProviderEnvironment": {
```

Comment:
We're trying very hard to move away from these codegen tests and use the new conformance test system instead. Could we remove this and add it to l2-provider-grpc-config test instead?